lib/ethpy/ethpy/hyperdrive/api.py

```diff
@@ -11,31 +11,23 @@
 from eth_account.signers.local import LocalAccount
 from eth_typing import BlockNumber
 from ethpy import EthConfig, build_eth_config
-from ethpy.base import (
-    BaseInterface,
-    async_smart_contract_transact,
-    get_account_balance,
-    initialize_web3_with_http_provider,
-    load_all_abis,
-    smart_contract_preview_transaction,
-    smart_contract_read,
-)
+from ethpy.base import (BaseInterface, async_smart_contract_transact,
```

Comment:
this is a new format that we don't use anywhere else in the code. I think it would be better to keep things consistent by leaving it alone. Not sure what changed on your end...